--- configure.orig	2016-11-04 08:52:44.000000000 +0800
+++ configure	2016-11-04 09:01:48.125368681 +0800
@@ -6152,17 +6152,8 @@
 		    V_FINDALLDEVS=fad-getad.c
 
 else
-
-		    #
-		    # We don't have the header - give up.
-		    # XXX - we could also fall back on some other
-		    # mechanism, but, for now, this'll catch this
-		    # problem so that we can at least try to figure
-		    # out something to do on systems with "getifaddrs()"
-		    # but without "ifaddrs.h", if there is something
-		    # we can do on those systems.
-		    #
-		    as_fn_error $? "Your system has getifaddrs() but doesn't have a usable <ifaddrs.h>." "$LINENO" 5
+                    # We don't have the header - use SIOCGIFCONF instead
+                    V_FINDALLDEVS=fad-gifc.c
 
 fi
 
