--- configure.orig	2017-07-23 07:37:54.000000000 +0800
+++ configure	2017-08-07 04:55:24.000000000 +0800
@@ -8057,9 +8057,9 @@
 	ac_fn_c_check_header_mongrel "$LINENO" "openssl/crypto.h" "ac_cv_header_openssl_crypto_h" "$ac_includes_default"
 if test "x$ac_cv_header_openssl_crypto_h" = xyes; then :
 
-		{ $as_echo "$as_me:${as_lineno-$LINENO}: checking for DES_cbc_encrypt in -lcrypto" >&5
-$as_echo_n "checking for DES_cbc_encrypt in -lcrypto... " >&6; }
-if ${ac_cv_lib_crypto_DES_cbc_encrypt+:} false; then :
+		{ $as_echo "$as_me:${as_lineno-$LINENO}: checking for DES_ncbc_encrypt in -lcrypto" >&5
+$as_echo_n "checking for DES_ncbc_encrypt in -lcrypto... " >&6; }
+if ${ac_cv_lib_crypto_DES_ncbc_encrypt+:} false; then :
   $as_echo_n "(cached) " >&6
 else
   ac_check_lib_save_LIBS=$LIBS
@@ -8073,27 +8073,27 @@
 #ifdef __cplusplus
 extern "C"
 #endif
-char DES_cbc_encrypt ();
+char DES_ncbc_encrypt ();
 int
 main ()
 {
-return DES_cbc_encrypt ();
+return DES_ncbc_encrypt ();
   ;
   return 0;
 }
 _ACEOF
 if ac_fn_c_try_link "$LINENO"; then :
-  ac_cv_lib_crypto_DES_cbc_encrypt=yes
+  ac_cv_lib_crypto_DES_ncbc_encrypt=yes
 else
-  ac_cv_lib_crypto_DES_cbc_encrypt=no
+  ac_cv_lib_crypto_DES_ncbc_encrypt=no
 fi
 rm -f core conftest.err conftest.$ac_objext \
     conftest$ac_exeext conftest.$ac_ext
 LIBS=$ac_check_lib_save_LIBS
 fi
-{ $as_echo "$as_me:${as_lineno-$LINENO}: result: $ac_cv_lib_crypto_DES_cbc_encrypt" >&5
-$as_echo "$ac_cv_lib_crypto_DES_cbc_encrypt" >&6; }
-if test "x$ac_cv_lib_crypto_DES_cbc_encrypt" = xyes; then :
+{ $as_echo "$as_me:${as_lineno-$LINENO}: result: $ac_cv_lib_crypto_DES_ncbc_encrypt" >&5
+$as_echo "$ac_cv_lib_crypto_DES_ncbc_encrypt" >&6; }
+if test "x$ac_cv_lib_crypto_DES_ncbc_encrypt" = xyes; then :
   cat >>confdefs.h <<_ACEOF
 #define HAVE_LIBCRYPTO 1
 _ACEOF
@@ -8102,7 +8102,7 @@
 
 fi
 
-		if test "$ac_cv_lib_crypto_DES_cbc_encrypt" = "yes"; then
+		if test "$ac_cv_lib_crypto_DES_ncbc_encrypt" = "yes"; then
 			for ac_header in openssl/evp.h
 do :
   ac_fn_c_check_header_mongrel "$LINENO" "openssl/evp.h" "ac_cv_header_openssl_evp_h" "$ac_includes_default"
